safe_web_api_playground/package.json

```diff
@@ -21,20 +21,19 @@
   "dependencies": {
     "babelify": "^7.3.0",
     "body-parser": "^1.17.2",
-    "bootstrap": "^4.1.3",
+    "bootstrap": "^4.3.1",
     "browserify": "^14.4.0",
     "codemirror": "^5.33.0",
     "envfile": "^2.1.1",
     "express": "^4.15.3",
-    "gulp": "^3.9.1",
+    "gulp": "^4.0.0",
     "gulp-concat": "^2.6.1",
     "gulp-image": "^2.9.0",
     "gulp-nodemon": "^2.2.1",
     "gulp-sass": "^3.1.0",
     "gulp-watch": "^4.3.11",
     "jquery": "^3.2.1",
     "npm-font-open-sans": "^1.1.0",
-    "safe_ux_guidelines": "https://github.com/maidsafe/safe_ux_guidelines.git",
```

Comment:
In addition to removing `safe_ux_guidelines`, those areas in the app which relied on it need to be refactored such as `index.html` and the local `scss` files.

Reply:
From what I observed in the gulpfile the `safe-styles` gulp task inserts the safe styles from the UX guidelines repo into the `static/scss` folder on which the `sass()` operation is performed to transform them into css files which are linked from `index.html`.